src/Illuminate/Support/Str.php

```diff
@@ -956,7 +956,7 @@ public static function replaceArray($search, $replace, $subject)
      * @param  string|iterable<string>  $replace
      * @param  string|iterable<string>  $subject
      * @param  bool  $caseSensitive
-     * @return string
+     * @return array|string|string[]
```

Comment:
`string[]` is a subtype of `array`, so should be deleted. It would be like writing `bool|false`. we just need `bool`.